CMake/resolve_dependency_modules/folly/folly-gflags-glog.patch

```diff
@@ -42,6 +42,6 @@
 +list(APPEND FOLLY_LINK_LIBRARIES ${glog_LIBRARY})
 +list(APPEND FOLLY_INCLUDE_DIRECTORIES ${glog_INCLUDE_DIR})
 +message(STATUS "glog_INCLUDE_DIR: ${gflags_LINRARY}")
```

Comment:
typo: `gflags_LINRARY`
Should we print glog library path as well?

Reply:
Oh a typo in the original patch. I suppose we could add it. This change was necessary to make the patch work with the updated folly-deps.cmake as it has added the fast_float right before our patch code.

Reply:
> Oh a typo in the original patch

Ah, I was see the diff of a diff.